typescript/api-client/projects/vcd/plugin-builders/README.md

```diff
@@ -69,6 +68,41 @@ npm i
 npm run build
 ```
 
+## For UI Plugins built for 10.2 (Zeus)
```

Comment:
I don't think should be included at this stage. Not sure about the visibility, but this is not yet released, so if it is visible you should exclude it from here.